index.bs

```diff
@@ -2663,6 +2680,10 @@ during credential generation.
     ::  This value indicates that the [=[RP]=] wants to receive the [=attestation statement=] as generated by the
         [=authenticator=].
 
+    :   <dfn>enterprise</dfn>
+    ::  This value indicates that the [=[RP]=] wants to receive an [=attestation statement=] that may include uniquely identifying information. This is intended for controlled deployments within an enterprise where the organization wishes to tie registrations to specific authenticators. User agents MUST NOT provide such an attestation unless user agent or authenticator configuration permits it for the requested [=RP ID=]. If not permitted, the user agent MUST treat this value the same as `direct`.
+
```

Comment:
Does It happen before or after the request is sent to the authenticator? We will plan to ask for permission before. 

If it happens after, because platform does not know whether authenticator will return enterprise attestation or not, doesn't treating it as `direct` in case user does not permit it disallows clients/platforms to change attestation to `none` or something anyway?

Do we really need a MUST here?

Reply:
on 2020-04-15 call:  (see minutes)  at minimum, we're going to delete the "If not permitted, the user agent MUST treat this value the same as `direct`." sentence.

Reply:
Have deleted that sentence.